src/pages/Register.tsx

```diff
@@ -58,8 +58,8 @@ const Register = () => {
 
   return (
     <form onSubmit={submitUserInfo} style={styles.container}>
-      <Typography variant="h1" sx={styles.title}>
-        Let's get to know a little more about you
+      <Typography sx={styles.title} variant="h1">
+        {`Let's get to know a little more about you`}
```

Comment:
is this something the formatter handles or did you manually edit this?

Reply:
It's actually not the formatter, it's `eslint-plugin-react`. It's because of the apostrophe character, [one of the rules](https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/no-unescaped-entities.md) prevents escape characters from appearing in the body of a tag without using special codes

![image](https://github.com/user-attachments/assets/5a477f5b-9fb9-4969-a54e-7a75db1c22de)